plugin-server/functional_tests/webhooks.test.ts

```diff
@@ -120,7 +120,9 @@ test.concurrent(`webhooks: fires zapier REST webhook`, async () => {
         const distinctId = new UUIDT().toString()
         const ts = new Date()
 
-        const organizationId = await createOrganization({ available_features: '{zapier}' })
+        const organizationId = await createOrganization({
+            available_product_features: `array ['{ "key": "zapier", "name": "zapier" }'::jsonb]`,
```

Comment:
Could use some help getting this to work...

Reply:
It keeps complaining about a malformed array literal : 
```
error: malformed array literal: "array ['{ "key": "zapier", "name": "zapier" }'::jsonb]"
```

Reply:
I also tried using jsonb_build_object

Reply:
That returns: 
``` 
detail: '"[" must introduce explicitly-specified array dimensions.',
```

Reply:
we're doing lots of wonky shenanigans with the generated SQL here (see inside createOrganization fn), easiest imo to just write the raw one in this case with the right ::jsonb coercion to make it work here, and not deal with postgres array literals at all (which don't work well with complicated jsonb conversion)

(see: https://stackoverflow.com/questions/69560635/problem-inserting-json-array-into-postgres-malformed-error )